web/html/src/components/hub/PeripheralsList.tsx

```diff
@@ -26,29 +26,31 @@ export class PeripheralsList extends React.Component<Props> {
         data="/rhn/manager/api/admin/hub/peripherals"
         identifier={(row: PeripheralListData) => row.id}
         initialSortColumnKey="fqdn"
+        defaultSearchField="fqdn"
         searchField={<SearchField filter={this.searchData} placeholder={t("Filter by FQDN")} />}
       >
         <Column columnKey="fqdn" comparator={Utils.sortByText} header={t("Peripheral FQDN")} cell={(row) => row.fqdn} />
         <Column
           columnKey="nChannelsSync"
-          comparator={Utils.sortByNumber}
+          sortable={false}
```

Comment:
This should be the default value, so setting it should not be necessary if I'm not mistaken.